pkgs/by-name/co/conan_1/package.nix

```diff
@@ -0,0 +1,194 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchPypi
+, git
+, python3
+}:
+
+let
+  python = python3.override {
+    packageOverrides = self: super: {
+      # conan v1.xx requires old semver instead of node-semver
```

Comment:
You need to override pydantic-extra-types too, it is not compatible with this old semver